File: android/src/main/java/au/com/crazybean/mobilex/kurir/modules/contacts/ContactsView.kt
package au.com.crazybean.mobilex.kurir.modules.contacts

import au.com.crazybean.foundation.navigator.Arguments
import au.com.crazybean.mobilex.kurir.data.model.User
import au.com.crazybean.mobilex.kurir.modules.base.View

interface ContactsView : View {
    fun showContacts(users: List<User>)
    fun showChat(arguments: Arguments)
    fun showSearch(arguments: Arguments)
    fun showEmpty()
}

File: ios/Kurir/modules/contacts/ContactsView.swift
//
//  ContactsView.swift
//  Kurir
//
//  Created by Loren on 19/9/19.
//  Copyright © 2019 Crazybean Studio. All rights reserved.
//

import Mobilex

protocol ContactsView {
    func showContacts(users: [User])
    func showChat(user: User)
    func showExplore()
    func showEmpty()
}
